util/security/path_traversal.go

```diff
@@ -17,6 +17,22 @@ func EnforceToCurrentRoot(currentRoot, requestedPath string) (string, error) {
 	return requestedDir + string(filepath.Separator) + requestedFile, nil
 }
 
+func SubtractRelativeFromAbsolutePath(abs, rel string) string {
```

Comment:
In Java there is a specific method to do this. Wonder if Go has similar.

Reply:
I searched for a bit and couldn't find any – at least not in the `filepath` package.